libp2p/Common.h

```diff
@@ -116,7 +116,7 @@ typedef std::pair<std::string, u256> CapDesc;
 typedef std::set<CapDesc> CapDescSet;
 typedef std::vector<CapDesc> CapDescs;
 
-struct PeerInfo
+struct PeerSessionInfo
```

Comment:
If this is to be code reviewed by external auditors, at least the .h files should have more comments. I assume an auditor would ask himself at least two questions: What is a PeersSessionInfo used for? What are "notes".